src/zkevm_specs/evm_circuit/execution/error_gas_uint_overflow.py

```diff
@@ -17,7 +21,7 @@ def error_gas_uint_overflow(instruction: Instruction):
 
     (
         is_call,
-        is_callcode,
+        _,
```

Comment:
We should put is_callcode in is_dynamic_gas too.

Reply:
fixed in [567055f](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/481/commits/567055f217dc65cb2e9d3c020db60f97f1ecc20e)